pkg/genericapiserver/genericapiserver.go

```diff
@@ -892,6 +894,25 @@ func (s *GenericAPIServer) InstallSwaggerAPI() {
 		},
 	}
 	swagger.RegisterSwaggerService(swaggerConfig, s.HandlerContainer)
+	openAPIConfig := openapi.Config{
```

Comment:
Re: this not being generic enough: Yes, spec.Info in particular I agree is not generic. For ex: we are planning to add a heapster server that will use genericapiserver. We want to expose swagger API there (and hence EnableSwaggerSupport will be true), but the title we want there will be different than "Kubernetes APIs". The version will be different too.
Same for openshift apiserver using genericapiserver library.

How about passing spec.Info as a different flag?

Re: having different flags for enabling 1.2 swagger spec and this 2.0 spec: I am fine keeping a single flag, but its not a lot of work to do if @smarterclayton or anyone feels strongly.